components/notebook-controller/.golang-ci.yaml

```diff
@@ -0,0 +1,45 @@
+# https://github.com/golangci/golangci-lint
```

Comment:
I would propose to add this file to a dedicated folder for CIs if possible. (The same for odh)

Reply:
Not possible ;( https://golangci-lint.run/usage/configuration/#config-file

Reply:
Ahh ok!